frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -147,20 +166,58 @@ pods:
         resource-set: sidecar-resources
       restore-schema:
         goal: FINISHED
-        cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/cqlsh -e "source 'container-path/snapshot/schema.cql'" node-${POD_INSTANCE_INDEX}-server.${FRAMEWORK_HOST} ${CASSANDRA_NATIVE_TRANSPORT_PORT}
+        cmd: >
+        {{#TASKCFG_ALL_CASSANDRA_ENABLE_TLS}}
+                ./bootstrap --resolve=false
+
+                export SSL_CERTFILE=$MESOS_SANDBOX/cqlsh.ca ;
+        {{/TASKCFG_ALL_CASSANDRA_ENABLE_TLS}}
+                ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/cqlsh {{{CASSANDRA_CQLSH_SSL_FLAGS}}} -e "source 'container-path/snapshot/schema.cql'" node-${POD_INSTANCE_INDEX}-server.${FRAMEWORK_HOST} ${CASSANDRA_NATIVE_TRANSPORT_PORT}
         resource-set: sidecar-resources
+        {{#TASKCFG_ALL_CASSANDRA_ENABLE_TLS}}
+        configs:
+          cqlshrc:
+            template: {{CONFIG_TEMPLATE_PATH}}/cqlshrc
+            dest: ./apache-cassandra-{{CASSANDRA_VERSION}}/conf/cqlshrc
+        transport-encryption:
+          - name: cqlsh
+            type: TLS
+        {{/TASKCFG_ALL_CASSANDRA_ENABLE_TLS}}
       restore-snapshot:
         goal: FINISHED
         cmd: >
+                set -ex ;
+                ./bootstrap --resolve=false ;
                 export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jre*/) ;
                 for f in $(find container-path/snapshot/ -maxdepth 1 -mindepth 1 -type d ! -name "system_*" ! -name "system") ; do
                     for t in $(find "$f" -maxdepth 1 -mindepth 1 -type d) ; do
-                        ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/sstableloader -f apache-cassandra-{{CASSANDRA_VERSION}}/conf/cassandra.yaml -d ${LOCAL_SEEDS} -p ${CASSANDRA_NATIVE_TRANSPORT_PORT} "$t" ;
+                        ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/sstableloader \
+                {{#TASKCFG_ALL_CASSANDRA_ENABLE_TLS}}
+                            --keystore node.keystore \
+                            --keystore-password notsecure \
+                            --truststore node.truststore \
+                            --truststore-password notsecure \
+                            --ssl-protocol TLSv1.2 \
```

Comment:
These options: e.g. `ssl-protocol` and `ssl-ciphers` are specified in the `cassandra.yml` file. Is there a way that we can use them here instead of duplicating them?

Reply:
For that matter, doesn't the `-f` option specify all these from the `cassandra.yaml` file?

Reply:
So I've tried to run this command without all the TLS options, just specifying a file but it didn't work so I'll keep it as it is without `-v` flag.